tests/phpunit/tests/rest-api/rest-plugins-controller.php

```diff
@@ -542,10 +542,29 @@ public function test_create_item_wdotorg_unreachable() {
 	}
 
 	/**
-	 * @ticket 50321
+	 * @ticket 50321x
 	 */
 	public function test_create_item_unknown_plugin() {
 		wp_set_current_user( self::$super_admin );
+		add_filter(
+			'pre_http_request',
+			static function() {
+				/*
+				 * Mocks the request to:
+				 * https://api.wordpress.org/plugins/info/1.2/?action=plugin_information&request%5Bslug%5D=alex-says-this-block-definitely-doesnt-exist&request%5Bfields%5D%5Bsections%5D=0&request%5Bfields%5D%5Blanguage_packs%5D=1&request%5Blocale%5D=en_US&request%5Bwp_version%5D=5.9
+				 */
+				return array(
+					'headers'  => array(),
+					'response' => array(
+						'code'    => 404,
+						'message' => 'Not Found',
+					),
+					'body'     => '{"error":"Plugin not found."}',
+					'cookies'  => array(),
+					'filename' => null,
+				);
+			}
+		);
 
 		// This will hit the live API.
```

Comment:
Let's remove this comment now.

Reply:
Removed in https://github.com/WordPress/wordpress-develop/pull/1868/commits/ad0d8b39c87ae556253cfccec2d8a31a9c377675.